index.js

```diff
@@ -1,4 +1,11 @@
 console.clear();
 
-const navClick = document.querySelector('[data-js="navicon"]');
-console.log(navClick);
+const bookmarkCard = document.querySelector('[data-js="bookmarkme"'); //creating the variable
```

Comment:
@ bookmarkme: Maybe name your data-js with caps or - or something else to make it easier to read :)